### DISCUSSION TITLE
Github ignoring npm tags now?

### DISCUSSION THREAD

**Discussion by jordlynn** [community/community#27495] [Actions]
An integration pipeline I used would tag a canary release every so often `npm publish --tag canary` for example. However starting yesterday it seems github "tags" all packages as `latest` and the `--tag` flag is ignored? 

Looking through the recent changes from github it seems there was an update to the security around npm publish, could this be affecting it?

**Comment by Niekteg** [comment]
I am experiencing the exact same issue at our organization. Every tags is always tagged as `latest`, other tags such as `beta` or `next` are not applied at all

**Comment by nicorikken** [comment]
Indeed, I've seen the same behavior. I noticed the [announcement of the v2 of the npm registry](https://github.com/orgs/community/discussions/21058) perhaps this is related, I'm not sure.

**Comment by juanibiapina** [comment]
Same for our organization. Even caused a small disruption when an incorrect version was used in production.

**Comment by elhedran** [comment]
In case this helps someone else - I had an error that looked similar to this.

I was publishing with `yarn npm publish --tag next` and was confused why when looking on the github webpage for the packages it was shown as `latest`.

However when I did a `yarn npm tag list` I noticed that there was no latest tag (just the next one).  As soon as I ensured a package was explicilty tagged latest, (`yarn npm tag add <package>@<correct version> latest`) the most recent package tagged next was no longer shown as `latest` in the github package page.  I suspect this may also be why some threads on this issue seem to indicate that things started working for users without an explicit fix - because it isn't how that `canary` or `beta` or `next` package is tagged, its that some other package in the mean time has been tagged `latest`.

May not be the same problem, but then maybe someone will come across this thread who did have the same issue I did and this may help them.
